source/components/inputs/input.ts

```diff
@@ -62,4 +69,14 @@ export class InputComponent<T> implements AfterViewInit, OnInit {
 			this.change.emit(value);
 		}
 	}
+
+	showLabel(): string {
+		return this.labelState = slide.show;
+	}
+
+	hideLabel(): string {
```

Comment:
Is there a better way we could name this, since you're only conditionally hiding the label? It seems strange to call `hideLabel()` and then have it just call `showLabel()` instead. Thoughts?